### PR TITLE
Added Vosk Adaptation Guide in Readme

### PR DESCRIPTION
**Pull request by devAyushDubey**
This PR contains Vosk Model Adaptation Guide, currently added in Readme.md

### CODE REVIEW

**Comment by cdhiraj40** [comment]
Hey @devAyushDubey, let's move this to another file inside the project, probably `vosk-model-adaption`
Other than that, everything looks good!

**Review by cdhiraj40 — CHANGES_REQUESTED** [review]
Hey @devAyushDubey, let's move this to another file inside the project, probably vosk-model-adaption
Other than that, everything looks good!

**Comment by SebaDro** [comment]
> Hey @devAyushDubey, let's move this to another file inside the project, probably vosk-model-adaption Other than that, everything looks good!

I also think that this guide should be part of a separate README. Maybe, create one, which is intended for voice command stuff inside the voice command module.

File: README.md [hunk 1]
```diff
@@ -94,6 +94,272 @@ The enviroCar App is licensed under the [GNU General Public License, Version 3](
...
+- Run `make`
+    ```sh
+    make -j 8
+    ```
+
+And Done 🎉,
+You can go ahead with the next steps.
+
+## Update Vosk Model
+There are many issues which can lead to bad accuracy from model mismatches to software bugs. See [accuracy](https://alphacephei.com/vosk/accuracy) guide for more detailed information on how to debug the accuracy problems. Once you figured out the reason is in the model mismatch you can try to update existing models to get better performance.
```
Comment (SebaDro) [inline README.md:163]: Be careful with copy&pasting whole sections from another documentation. You'd rather consider a reqrite or add appropriate credits.

**Review by SebaDro — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SebaDro) [reply README.md:163]: I've noticed several other sections that have been copy and pasted. Please, revise your whole documentation. If sections does not provide any new instructions than the original one from the [Vosk homepage](https://alphacephei.com/vosk/adaptation), please remove these sections and just link to the original guide.

**Review by SebaDro — APPROVED** [review]
Looks good to me. I will merge it, as soon as @cdhiraj40 also approves.

**Review by cdhiraj40 — APPROVED** [review]
Looks good to me! Ready to merge :)

**Pull request merged**